R/io.R

```diff
@@ -20,3 +20,58 @@ make_temporary_file <- function(content,extension) {
 read_temporary_file <- function(temp){
   return(readLines(temp))
 }
+
+#' This function creates a sample file
+#'
+#' @param save_filepath The path of the file that is being saved
+#' @param body The contents of the file
+#' @return file with specified path and body
+#' @export
+io_make_sample_file <- function(save_filepath,body) {
+
+  error <- system2('echo',
+                   args = c("\"print('hello world!')\"",
+                            '>',
+                            file_path),
+                   stdout = TRUE,
+                   stderr = FALSE)
+
+  # Create cmd
+  cmd <- paste("echo", shQuote(body), ">", shQuote(save_filepath))
+
+  # Execute the command using system2
+  result <- system2(cmd, stdout = TRUE, stderr = FALSE)
+
+
+}
+
+#' This function creates a new folder
+#'
+#' @param folderName Name of the folder you are creating
+#' @param body The contents of the folder, what files it will consist of
+#' @return The new folder with its updated contents
+#' @export
+io_create_folder <- function(folderName, body) {
+
+  file_path <- file.path(folder_path,"hello.R")
+
+  #echo "print('hello world!')" >  path/to/folder/hello.R
+  error <- system2('echo',
+                   args = c("\"print('hello world!')\"",
+                            '>',
+                            file_path),
+                   stdout = TRUE,
+                   stderr = FALSE)
```

Comment:
I'm not sure what is going on here, this looks the same code as above, minus the correct command which should have been mkdir!